modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_tags_selector/AssetTagsSelector.es.js

```
@@ -136,17 +149,7 @@ class AssetTagsSelector extends Component {
 		this.selectedItems = event.data.selectedItems;
 		this.tagNames = this._getTagNames();
 
-		if (this.addCallback) {
-			window[this.addCallback](event.data.item);
-		}
-
-		this.emit(
-			'itemAdded',
-			{
-				item: event.data.item,
-				selectedItems: this.selectedItems
-			}
-		);
+		this._notifyItemsChanged('itemAdded', this.addCallback, item);
```

`item` is not defined here. This should throw an exception.